app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,57 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all_tasks
+  end
+
+  def show
+    @task = Task.find(params[:id])
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = Task.new(create_params[:task])
+    @task.save
+    redirect_to "/"
+  end
+
+  def delete
+    @task = Task.find(params[:id])
+    @task.delete #destroy without save
```

Comment:
The distinction between `delete` and `destroy` is actually more subtle than that. Delete still deletes the row from the database, but it doesn't execute any callbacks or delete associated models. Destroy executes callbacks and deletes any associations that have `:dependent => :destroy` or similar configuration.